repo2docker/buildpacks/julia/julia_project.py

```diff
@@ -1,5 +1,8 @@
 """Generates a Dockerfile based on an input matrix for Julia"""
+from functools import cmp_to_key
 import os
+import requests
+import semver
```

Comment:
Should this be `from . import semver` and a few lines further down with the other imports from within the package?

Reply:
I changed it to import python package `semver` from local module `semver`.